sellotape/sellotape_dj/chat/templates/chat/room.html

```diff
@@ -0,0 +1,94 @@
+{% load static %}
+<html>
```

Comment:
Is there a specific reason not to make use of the base.html template you guys implemented?